sdk/ml/azure-ai-ml/azure/ai/ml/entities/_registry/registry.py

```diff
@@ -83,7 +83,12 @@ def dump(self, dest: Union[str, PathLike, IO[AnyStr]]) -> None:
 
     def _to_dict(self) -> Dict:
         # pylint: disable=no-member
-        return RegistrySchema(context={BASE_PATH_CONTEXT_KEY: "./"}).dump(self)
+        schema = RegistrySchema(context={BASE_PATH_CONTEXT_KEY: "./"})
+        self.replication_locations = self.region_details
+        #if self.replication_locations and len(self.replication_locations) > 0:
```

Comment:
Why are we keeping this commented section?

Reply:
Dropping this PR - the changes were incorporated into a separate, larger PR that's already been merged.